tmk_core/protocol/lufa/adafruit_ble.cpp

```diff
@@ -38,7 +38,7 @@
 
 #ifdef SAMPLE_BATTERY
 #    ifndef BATTERY_LEVEL_PIN
-#        define BATTERY_LEVEL_PIN 7
+#        define BATTERY_LEVEL_PIN B5
```

Comment:
I don't follow how `7` becomes `B5` here – did Adafruit change the pinout on this module at some point?

Reply:
I think the LUT used by `analogRead()` was incorrect (specifically, the mux value `0x24`):
https://github.com/qmk/qmk_firmware/pull/7191/files#diff-b241d88b4050a11d27f133d58f8c40abL31